Prototype/Components/questionnaire/questionnaire.html

```diff
@@ -0,0 +1,73 @@
+<div class="cmp-questionnaire">
+  <div class="cmp-questionnaire__container">
+      <img class="cmp-questionnaire__image"
+        src="../../images/women-with-laptop.png"
+        alt="Women with daughter"
+      />
+    <form class="cmp-questionnaire__form">
+      <div class="cmp-questionnaire__title">
+        <div class="cmp-questionnaire__heading">Have a Question?</div>
+        <div class="cmp-questionnaire__content">
+          We're here to answer!Reach out throught the quick form below,on social
+          media,or use our live chat to talk to someone directly,now!
+        </div>
+      </div>
+      <div class="cmp-questionnaire__fields">
+         <div class="cmp-questionnaire__inputfields">
+            <div class="cmp-questionnaire__inputfields__firstlabel">
+                <label class="cmp-questionnaire__inputfields__label">First Name</label>
+                <input class="cmp-questionnaire__inputfields__textfield" type="text" placeholder="Enter Your First Name" id="fname" name="fname" required/>
+            </div>
+            <div class="cmp-questionnaire__inputfields__lastlabel">
+                <label class="cmp-questionnaire__inputfields__label">Last Name</label>
+                <input class="cmp-questionnaire__inputfields__textfield" type="text" placeholder="Enter Your Last Name" id="lname" name="lname" required/>
+            </div> 
+         </div>
+       <div class="cmp-questionnaire__inputfields">
+        <div class="cmp-questionnaire__inputfields__firstlabel">
+            <label class="cmp-questionnaire__inputfields__label">Phone</label>
+            <input class="cmp-questionnaire__inputfields__textfield" type="tel" placeholder="Enter Phone Number" name="phNumber" />
+            
+        </div> 
+        <div class="cmp-questionnaire__inputfields__lastlabel">         
+            <label class="cmp-questionnaire__inputfields__label">Email</label>
+            <input class="cmp-questionnaire__inputfields__textfield" type="email" placeholder="Enter Your Email Address" name="formEmail" required/>
+        </div>
+       
+       </div>
+       <div class="cmp-questionnaire__inputfields">
+        <div class="cmp-questionnaire__inputfields__lastlabel">
+            <label class="cmp-questionnaire__inputfields__label">Message</label>
+            <div class="cmp-questionnaire__inputfields__comments">Enter Your Questions or Comments</div>
+            <input class="cmp-questionnaire__inputfields__message"
+              name="message"/>
+        </div>
+       </div>
+       <div class="cmp-questionnaire__inputfields__contactsection">
+        <div class="cmp-questionnaire__inputfields__contact">
+            <div>
+                <label class="cmp-questionnaire__inputfields__label">Preferred Contact Method</label>
+            </div>
+            <div >         
+                <label class="cmp-questionnaire__inputfields__contact" for="phone"> 
```

Comment:
@chintanpatelrightpoint this is not accessible checkbox if you want to make custom accessible checkbox using css then add for attribute in label and id tag in checkbox and give the same value. You can refer to this blog https://medium.com/claritydesignsystem/pure-css-accessible-checkboxes-and-radios-buttons-54063e759bb3
Cc:- @jvaishnavrp  @msharmaRP

Reply:
Here's an example using `appearance: none;` and `&:checked` on the input itself so position doesn't need absolute/relative and should still be fully accessible:
https://codepen.io/jwetmore-hoodoo/pen/eYwJBvB

Also on accessibility, if you wrap the text content and an `input` with a `label` like I've done in this Codepen, you don't need to use `for` on anymore.

Sometimes that's not possible due to where the design wants the label text though, so if you have to separate them, make sure to use `for`.

Either way, you'll want a `name` attribute on the `input` anyway so the processor of the data knows what field is which.